src/locales/lang/en.ts

```diff
@@ -152,6 +152,7 @@ const msg = {
   text: "Text",
   query: "Query",
   postgreSQL: "PostgreSQL",
+  endpointTips: "The table shows up to 20 pieces of data.",
```

Comment:
```suggestion
  endpointTips: "The table shows up to 20 endpoints.",
```

Could we move this text next to the search bar?

Reply:
Done.

<img width="1045" alt="3" src="https://user-images.githubusercontent.com/20871783/189132940-b05bd28e-db5b-4dc9-aa24-f47e78a01d7d.png">